src/main/java/com/wise23/chariteed/controller/PatientController.java

```diff
@@ -10,7 +11,7 @@ public class PatientController {
 
     PatientService patientService = new PatientService();
 
-    @GetMapping("/patient/{id}")
+    @GetMapping("/admin/patient/{id}")
     public String getPatientById(@PathVariable("id") String id) {
         Patient patient = patientService.client.read().resource(Patient.class).withId(id).execute();
         return patientService.fhirContext.newJsonParser().setPrettyPrint(true).encodeResourceToString(patient);
```

Comment:
Why not return object in Json format?
If this method is used for fetching data, I would suggest to return simple json object.
Otherwise, I would create a separate thymeleaf template and customize it.

Reply:
we have not put our focus on the format of patient data yet, just gave it a try of this test server

Reply:
We're not using this at the moment (I think it's some left over code), so I left it as it is and maybe we can remove it.